conftest.py

```diff
@@ -89,6 +92,28 @@ def mock_lsf_version(monkeypatch, request):
 }
 
 
+# Overriding cleanup method from distributed that has been added to the loop
+# fixture, because it just wipe the Main Loop in our tests, and dask-jobqueue is
+# not ready for this.
+# FIXME
+@pytest.fixture
+def cleanup():
+    dask_jobqueue.config.reconfigure()
+    yield
+
+
+# Overriding distributed.utils_test.reset_config()  method because it reset the
+# config from ou tests.
```

Comment:
Our

Reply:
Woops, I was too eager to merge this, I left a typo, thanks for the proof read! I hope anyway that this text will soon disappear.